pkg/sources/gcppubsub/googleapis-serviceentry.yaml

```diff
@@ -24,3 +24,42 @@ spec:
     name: https
     protocol: HTTPS
   location: MESH_EXTERNAL
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: metadata-server-k8s
```

Comment:
This should probably be `metadata-server-gce`, and we should add a note that this sample only works on GKE or k8s clusters on GCE.

Reply:
the address http://metadata/ was a surprise to all of us; it's the address that's only used when on GKE. Normally the address is http://metadata.google.internal/